src/temple/front/chains.ts

```diff
@@ -4,13 +4,18 @@ import { TEZOS_MAINNET_CHAIN_ID } from 'lib/temple/types';
 import type { StoredTezosNetwork, StoredEvmNetwork } from 'temple/networks';
 import type { TempleChainKind } from 'temple/types';
 
+import { BlockExplorer } from './block-explorers';
 import { useAllTezosChains, useAllEvmChains } from './ready';
 
-interface ChainBase {
+export interface ChainBase {
   rpcBaseURL: string;
   name: string;
   nameI18nKey?: TID;
   disabled?: boolean;
+  allBlockExplorers: BlockExplorer[];
+  activeBlockExplorer?: BlockExplorer;
+  mainnet: boolean;
```

Comment:
Ok, no problem. However, the first principle cannot be applied to chains from `viem/chains` because lots of mainnet chains there have `testnet` flag equal to `undefined`